### PR TITLE
💅 landing top collection asset hub

### LINKED ISSUE #7976: Assethubs top collections landing page section
we are missing top collection section on assethubs landing page 
image

needs indexer

### PR DESCRIPTION
**Pull request by roiLeo**
- close #7976 
- code fast, refactor later

## AHP
Screenshot 2023-12-15 at 11-42-01 KodaDot - NFT Market Explorer

## AHK
Screenshot 2023-12-15 at 11-48-57 KodaDot - NFT Market Explorer

### CODE REVIEW

**Comment by prury** [comment]
@roiLeo if i switch between let's say kusama and kusamahub twice, the table will stop updating itself:

https://github.com/kodadot/nft-gallery/assets/36627808/8fb16787-973f-45ac-b30f-96dadc1beec2

**Comment by roiLeo** [comment]
> @roiLeo if i switch between let's say kusama and kusamahub twice, the table will stop updating itself:

legit, nice catch! refactor has been made.

note that days filter don't work for ah (& maybe other chains?)

File: queries/subsquid/general/topCollections.graphql [hunk 1]
```diff
@@ -0,0 +1,17 @@
...
+    where: { volume_gt: "0" }
+  ) {
+    id
+    name
+    image
+    media
+    volume
+    floor
+    nftCount
+  }
```
Comment (vikiival) [inline queries/subsquid/general/topCollections.graphql:16]: supply?

**Review by vikiival — COMMENTED** [review]
~

**Comment by prury** [comment]
> 
>refactor has been made.

i think the problem still persists

**Comment by yangwao** [comment]
switching between AHP <-> AHK keeps DOT but shows AHK collections, can be fixed later tho 

image


floor is not there? 

image

**Comment by yangwao** [comment]
but works ok on first load

**Pull request merged**